### PR TITLE
Add reader for Stichotheque Portuguese corpus

### PR DESCRIPTION
**Pull request by aitordiaz**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -74,6 +74,8 @@ Available corpora (version 1.1.0)
   11  Corpus of            cs      4100M    66428  12636867  stanza         CC-BY-SA
       Czech Verse                                            line
       (czverse)                                              word
+  12  Stichotheque         pt      11,8M     1702    168411  stanza         Unkwown
```
Comment (versae) [inline README.rst]:
```suggestion
  12  Stichotheque         pt      11.8M     1702    168411  stanza           12  Stichotheque         pt      11,8M     1702    168411  stanza         Unkwown
```
Is it really `Unkwown`?

File: src/averell/corpora.yaml [hunk 2]
```diff
@@ -148,3 +148,16 @@
...
         - line
         - word
+- name: Stichotheque Portuguese
+  properties:
+    slug: stichopt
+    license: Unkwown
+    language: pt
+    reader: averell.readers.stichotheque
+    url: https://gitlab.com/stichotheque/stichotheque-pt/-/archive/master/stichotheque-pt-master.zip
+    size: 11,8M
```
Comment (versae) [inline src/averell/corpora.yaml]:
```suggestion
    size: 11.8M
```

File: src/averell/readers/stichotheque.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    expressions.
+    Then, we iterate over the poem text and we look for each stanza and line
+    data.
+    :param xml_file: Path for the xml file
+    :return: Poem python dict with the data obtained
+    """
+    tree = ETree.parse(xml_file)
+    root = tree.getroot()
+    corpus_name = xml_file.parts[-4]
+    manually_checked = False
```
Comment (versae) [inline src/averell/readers/stichotheque.py:22]: Is it all automatically generated? Bummer.

**Review by versae — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (aitordiaz) [reply README.rst]: They wrote this:

> We do not have a formal license yet, but if you would like to use the files provided by Stichotheque, these are the informal terms:
> - You can use the files for research and education, but not for commercial purposes.
> - If you write an article that uses these files in some way, we ask you to cite one of our publications.

So I don't know which license to choose...

File: src/averell/readers/stichotheque.py (hunk 3 above)
Comment (aitordiaz) [reply src/averell/readers/stichotheque.py:22]: Yep, when the `meter` tag appears it is within the Aoidos Namespace.

Aoidos is their automatic scansion tool.

File: src/averell/corpora.yaml (hunk 2 above)
Comment (aitordiaz) [reply src/averell/corpora.yaml]: 🤯. Sorry, changed

File: README.rst (hunk 1 above)
Comment (versae) [reply README.rst]: That sounds a lot like CC-BY or an edu license. Let's keep it as Unknown. We need add links to their source projects at some point.

Comment (aitordiaz) [reply README.rst]: Wich is the best option?

Comment (aitordiaz) [reply README.rst]: > Wich is the best option?

Sorry I wrote this without refreshing the page. OK, Unknown.

**Review by versae — DISMISSED** [review]

File: README.rst (hunk 1 above)
Comment (versae) [reply README.rst]: Please, it's `11.8M` not `11,8M`

File: src/averell/corpora.yaml [hunk 4]
```diff
@@ -148,3 +148,16 @@
...
         - line
         - word
+- name: Stichotheque Portuguese
+  properties:
+    slug: stichopt
+    license: Unkwown
+    language: pt
+    reader: averell.readers.stichotheque
+    url: https://gitlab.com/stichotheque/stichotheque-pt/-/archive/master/stichotheque-pt-master.zip
+    size: 11.8M
```
Comment (aitordiaz) [inline src/averell/corpora.yaml:158]: I changed it in the second commit, I don't know why it is not displayed on the main page of the PR

Comment (aitordiaz) [reply src/averell/corpora.yaml:158]: Oh sorry, its in the other file

File: README.rst (hunk 1 above)
Comment (aitordiaz) [reply README.rst]: My fault, done!

**Review by versae — APPROVED** [review]

**Pull request merged**